src/vs/workbench/contrib/positronHoloViews/test/browser/positronHoloViewsService.test.ts

```diff
@@ -0,0 +1,133 @@
+import { IPositronNotebookOutputWebviewService } from 'vs/workbench/contrib/positronOutputWebview/browser/notebookOutputWebviewService';
+import { PositronNotebookOutputWebviewService } from 'vs/workbench/contrib/positronOutputWebview/browser/notebookOutputWebviewServiceImpl';
+import { WebviewPlotClient } from 'vs/workbench/contrib/positronPlots/browser/webviewPlotClient';
+import { IWebviewService } from 'vs/workbench/contrib/webview/browser/webview';
+import { WebviewService } from 'vs/workbench/contrib/webview/browser/webviewService';
+import { LanguageRuntimeSessionMode, RuntimeOutputKind } from 'vs/workbench/services/languageRuntime/common/languageRuntimeService';
+import { IRuntimeSessionService } from 'vs/workbench/services/runtimeSession/common/runtimeSessionService';
+import { TestLanguageRuntimeSession } from 'vs/workbench/services/runtimeSession/test/common/testLanguageRuntimeSession';
+import { TestRuntimeSessionService } from 'vs/workbench/services/runtimeSession/test/common/testRuntimeSessionService';
+import { workbenchInstantiationService } from 'vs/workbench/test/browser/workbenchTestServices';
+
+class TestNotebookService implements Partial<INotebookService> {
+	getRenderers(): INotebookRendererInfo[] {
+		return [];
+	}
+
+	getPreferredRenderer(_mimeType: string): NotebookOutputRendererInfo | undefined {
+		// Doesn't matter what the renderer is, just that it exists.
+		return <NotebookOutputRendererInfo>{
+			id: 'positron-ipywidgets',
+			extensionId: new ExtensionIdentifier('vscode.positron-ipywidgets'),
+		};
+	}
+
+	*getStaticPreloads(_viewType: string): Iterable<INotebookStaticPreloadInfo> {
+		// Yield nothing.
+	}
+}
+
+suite('Positron - PositronHoloViewsService', () => {
+	const disposables = ensureNoDisposablesAreLeakedInTestSuite();
+
+	let positronHoloViewsService: PositronHoloViewsService;
+	let runtimeSessionService: TestRuntimeSessionService;
+	let notebookEditorService: INotebookEditorService;
+
+	setup(() => {
+		const instantiationService = workbenchInstantiationService(undefined, disposables);
+		instantiationService.stub(INotebookRendererMessagingService, disposables.add(instantiationService.createInstance(NotebookRendererMessagingService)));
+		notebookEditorService = disposables.add(instantiationService.createInstance(NotebookEditorWidgetService));
+		instantiationService.stub(INotebookEditorService, notebookEditorService);
+		instantiationService.stub(INotebookService, new TestNotebookService());
+		instantiationService.stub(IWebviewService, disposables.add(new WebviewService(instantiationService)));
+		instantiationService.stub(IPositronNotebookOutputWebviewService, instantiationService.createInstance(PositronNotebookOutputWebviewService));
+		runtimeSessionService = disposables.add(new TestRuntimeSessionService());
+		instantiationService.stub(IRuntimeSessionService, runtimeSessionService);
```

Comment:
We could add our own function like `workbenchInstantiationService` that does this if its repeated too often, not strictly necessary.

Reply:
I attempted this real quick and found I was creating just about as much code. Next test suite we need to make I bet will make it clear how to best abstract it.